packages/smooth_app/.vscode/launch.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Okay with committing this as it's very likely useful for other VSVode users but normally should be in a different PR